tests/SemVerAnalyzer.Tests/IntegrationTests.cs

```diff
@@ -118,5 +118,18 @@ public async Task OmitDisclaimer()
 
 			PAssert.IsTrue(() => !report.Contains("*This is a sanity check"));
 		}
+
+		[Fact]
+		public async Task OmitDisclaimerFalse()
+		{
+			var command = new CompareCommand {
+				Assembly = "Local.dll",
+				PackageName = "Major"
+			};
+
+			var report = await _runner.Compare(command);
+
+			PAssert.IsTrue(() => report.Contains("*This is a sanity check"));
```

Comment:
only concern is `*This is a sanity check` is duplicated, this could be a theory to reuse the same test, but this LGTM